builder/scaleway/builder.go

```diff
@@ -19,6 +21,8 @@ import (
 // BuilderId is the unique id for the builder
 const BuilderId = "hashicorp.scaleway"
 
+var acceptanceTests = flag.Bool("acceptance tests", os.Getenv("PACKER_ACC") == "1", "Run acceptance tests")
```

Comment:
```suggestion
var acceptanceTests = flag.Bool("run-acceptance-tests", os.Getenv("PACKER_ACC") == "1", "Run acceptance tests")
```

Should it be all attached?

Reply:
I'm not sure how does the option present itself in the test binary?